musabase/cgiar_banana_ontology.obo

```diff
@@ -5706,6 +5706,229 @@ namespace: BananaTrait
 def: "A quick field evaluation of the raw pulp to determine acceptability based on yellow color using Mbwazirume as a standard control." []
 is_a: CO_325:1000006 ! Quality_traits
 
+[Term]
+id: CO_325:0003004
+name: weevil damage assessment
+namespace: BananaTrait
+def: "Weevil damage assesssment" []
+is_a: CO_325:1000002 ! Biotic_stress
+
+[Term]
+id: CO_325:0003005
+name: Diameter at the stellar region in cm after weevil introduction
+namespace: BananaTrait
+def: "slice the corm measure the diameter using a tape (cm) 35 and 66 days after weevil introduction (experiement done in potted plant) In cm" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010155 ! corm size - measurement
+relationship: variable_of CO_325:0100008 ! cm
+
+[Term]
+id: CO_325:0003006
+name: Diameter at 5 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+def: "slice the corm measure the diameter using a tape (cm) 35 and 66 days after weevil introduction (experiement done in potted plant) In cm" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010155 ! corm size - measurement
+relationship: variable_of CO_325:0100008 ! cm
+
+[Term]
+id: CO_325:0003007
+name: Diameter at 10 cm from the stellar region after weevil introduction
+namespace: BananaTrait
+def: "slice the corm measure the diameter using a tape (cm) 35 and 66 days after weevil introduction (experiement done in potted plant) In cm" []
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010155 ! corm size - measurement
+relationship: variable_of CO_325:0100008 ! cm
+
+[Term]
+id: CO_325:0003008
+name: Inner damage percentage at the stellar region after weevil introduction
+namespace: BananaTrait
+is_a: CO_325:1000024 ! Variable
+relationship: variable_of CO_325:0003004 ! weevil damage assessment
+relationship: variable_of CO_325:0010155 ! corm size - measurement
+relationship: variable_of CO_325:0100046 ! %
+
+[Term]
+id: CO_325:0003009
```

Comment:
there is no 'outer damage percentage at the stellar region'. Has it been forgotten? Also check this variable before adding: 
`id: CO_325:0000844
name: Corm outer diameter after weevils introduction in cm`